deployment/gce/lily/notifier.env

```diff
@@ -0,0 +1,16 @@
+# Path for lily state should be on a persisted volume
```

Comment:
need `LILY_BLOCKSTORE_CACHE_SIZE` and `LILY_STATESTORE_CACHE_SIZE` set to 500000.